packages/devtools-utils/src/tests/worker-utils.js

```diff
@@ -74,4 +72,55 @@ describe("worker utils", () => {
       error: new Error("failed"),
     });
   });
-})
+});
+
+it("streams a task", async () => {
+  jest.useRealTimers();
+
+  const postMessageMock = jest.fn();
+
+  const worker = {
+    postMessage: postMessageMock
+  };
+
+  function makeTasks() {
+    return [
+      {
+        callback: () => new Promise(resolve => setTimeout(() => resolve(1), 50))
+      },
+      {
+        callback: () => new Promise(resolve => setTimeout(() => resolve(2), 50))
+      }
+    ];
+  }
+
+  const workerHandler = streamingWorkerHandler(
+    { makeTasks },
+    { timeout: 25 },
+    worker
+  );
+
+  const id = 1;
+  const task = workerHandler({ data: { id, method: "makeTasks", args: [] }});
+  await task;
+
+  expect(postMessageMock.mock.calls.length).toBe(4);
+  expect(postMessageMock.mock.calls[0][0]).toEqual({
+    id,
+    status: "start"
+  });
+  expect(postMessageMock.mock.calls[1][0]).toEqual({
+    id,
+    status: "pending",
+    data: [2]
+  });
+  expect(postMessageMock.mock.calls[2][0]).toEqual({
+    id,
+    status: "pending",
+    data: [1]
+  });
+  expect(postMessageMock.mock.calls[3][0]).toEqual({
+    id,
+    status: "done"
+  });
+});
```

Comment:
Haha - can you explain what this is testing?

I expected to see call reaults stream in over time.